src/vue/modules/create-asset-sale/components/create-asset-sale-form.vue

```diff
@@ -0,0 +1,737 @@
+  props: {
+    accountId: {
+      type: String,
+      required: true,
+    },
+  },
+  data () {
+    return {
+      form: {
+        information: {
+          name: '',
+          code: '',
+          policies: [],
+          assetType: '',
+          formType: {},
+          terms: null,
+        },
+        saleInformation: {
+          name: '',
+          startTime: '',
+          endTime: '',
+          maturityDate: '',
+          softCap: '',
+          hardCap: '',
+          annualReturn: '',
+          quoteAssets: [],
+        },
+        shortBlurb: {
+          saleLogo: null,
+          shortDescription: '',
+          youtubeId: '',
+          description: '',
+        },
+      },
+      currentStep: 1,
+      STEPS,
+      kvAssetTypeKycRequired: '',
+      MIN_AMOUNT: config.MIN_AMOUNT,
+      MAX_AMOUNT: config.MAX_AMOUNT,
+      config,
+      ASSET_SUBTYPE,
+      DOCUMENT_TYPES,
+      ASSET_POLICIES,
+      CODE_MAX_LENGTH,
+      NAME_MAX_LENGTH,
+      DESCRIPTION_MAX_LENGTH,
+    }
+  },
+  validations () {
+    return {
+      form: {
+        information: {
+          name: {
+            required,
+            maxLength: maxLength(NAME_MAX_LENGTH),
+          },
+          code: {
+            required,
+            maxLength: maxLength(CODE_MAX_LENGTH),
+          },
+          assetType: {
+            required,
+          },
+          formType: {
+            required,
+          },
+        },
+        saleInformation: {
+          startTime: {
+            required,
+          },
+          endTime: {
+            required,
+            minDate: minDate(this.form.saleInformation.startTime ||
+              moment().toString()),
+          },
+          maturityDate: {
+            required: requiredUnless(function () {
+              return this.form.information.formType.value ===
+                ASSET_SUBTYPE.bondCreation
+            }),
+            minDate: minDate(
+              this.form.information.formType.value === ASSET_SUBTYPE.bond
+                ? this.form.saleInformation.endTime : 0),
+          },
+          softCap: {
+            required,
+            amountRange: amountRange(this.MIN_AMOUNT, this.MAX_AMOUNT),
+          },
+          hardCap: {
+            required,
+            amountRange: amountRange(this.form.saleInformation.softCap,
+              this.MAX_AMOUNT),
+          },
+          annualReturn: {
+            required,
+            amountRange: amountRange('0',
+              this.MAX_AMOUNT),
+          },
+          quoteAssets: {
+            requiredAtLeastOne,
+          },
+        },
+        shortBlurb: {
+          saleLogo: {
+            required,
+          },
+          shortDescription: {
+            required,
+            maxLength: maxLength(DESCRIPTION_MAX_LENGTH),
+          },
+        },
+      },
+    }
+  },
+  computed: {
+    formTypes () {
+      return [
+        {
+          label: globalize('create-asset-sale-form.bond-creation'),
```

Comment:
Pass only translation key, don't use `globalize` outside any `<template>`. Select-field will use `globalize` instead of you